src/cli/commands/test/iac-local-execution/types.ts

```diff
@@ -1,14 +1,35 @@
 import { SEVERITY } from '../../../../lib/snyk-test/common';
 import { IacFileInDirectory } from '../../../../lib/types';
 
-export type IacFileMetadata = IacFileInDirectory;
-export interface IacFileData extends IacFileMetadata {
-  jsonContent: Record<string, any>;
+export interface IacFileData extends IacFileInDirectory {
   fileContent: string;
+}
+export const VALID_FILE_TYPES = ['tf', 'json', 'yaml', 'yml'];
+
+export interface ParsedIacFile extends IacFileData {
+  jsonContent: Record<string, any>;
```

Comment:
I think we can get away with `unknown` here and avoid the `any` as we never read this in TS land right?
```suggestion
  jsonContent: Record<string, unknown>;
```